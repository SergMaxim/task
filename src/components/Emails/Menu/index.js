import React, { Component } from 'react';
import Styles from './styles.scss';
import { func, object } from 'prop-types';

export default class Menu extends Component {

    /*static propTypes = {
        menu:     object.isRequired,
        openForm: func.isRequired
    };*/
    render () {
        return (
            <section className = { Styles.menu }>
                <div className = { Styles.button }>Compose Mail</div>
            </section>
        );
    }
}
