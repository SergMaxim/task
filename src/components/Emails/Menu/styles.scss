.menu{
  width: 20%;
  position: relative;

  .button{
    width: 80%;
    height: 20px;
    line-height: 20px;
    border-radius: 50px;
    color: #fff;
    cursor: pointer;
    background-color: #00b1ef;
    margin: 0 auto;
    text-align: center;
    padding: 10px;
    box-sizing: content-box;
    margin-top: 10px;
  }
}