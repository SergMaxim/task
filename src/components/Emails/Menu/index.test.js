import React from 'react';
import { configure, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import dom from 'react-test-renderer';

// Instruments
import Menu from './';

configure({ adapter: new Adapter() });

const renderTree = dom.create(<Menu />).toJSON();

describe('Menu component', () => {
    test('Menu component should correspond to its snapshot counterpart', () => {
        expect(renderTree).toMatchSnapshot();
    });
});
