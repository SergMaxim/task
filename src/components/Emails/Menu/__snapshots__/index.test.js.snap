// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component Menu component should correspond to its snapshot counterpart 1`] = `
<section
  className="menu"
>
  <div
    className="button"
  >
    Compose Mail
  </div>
</section>
`;
